src/Illuminate/Notifications/RoutesNotifications.php

```diff
@@ -20,6 +20,8 @@ public function notify($instance)
 
     /**
      * Get the notification routing information for the given driver.
+     * In case of an empty return value, the notification will
+     * not be sent.
```

Comment:
> the brain somehow expects an error message if the email is not delivered, at least mine did in this case.

What if you have a system that configure a notification to be send via sms, mail and push notification but at the same time user have the ability to either allow which channel to be used.

If you were throwing exception, this would be bad doesn't it?